client/scripts/views/pages/threads/index.ts

```diff
@@ -39,16 +51,16 @@ export const newThread = (
   readOnly?: boolean
 ) => {
   if (!form.title) {
-    return ({ title: 'Title cannot be blank' });
+    return ({ title: NewThreadErrors.NoTitle });
```

Comment:
Any particular reason we wouldn't return an object with an `error` property for all of these?

Reply:
Yup, it's cuz newthread submissions are set up to handle error submission as cause-specific key-val pairs. There's definitely a way to rewrite it for consistency across the codebase; it just seems out of scope here? Do we have a house style or a default pattern we should go through and institute uniformly?